.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Java CI with Gradle
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+      - name: Build with Gradle
```

Comment:
I think the build because there is no config for `signArchives` task. So, we can ignore this task build use this command `./gradlew build --exclude-task signArchives`.

Reply:
Thank you. We'll have a separate action for building + publishing. This one is for fixing failed CI